Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoEmptyBlockRule.swift

```diff
@@ -54,6 +55,42 @@ struct NoEmptyBlockRule: Rule {
             Example("""
             while i < 10 { /* do something */ }
             """),
+            Example("""
+            var flag = true {
+                willSet {}
+            }
+            """, configuration: ["disabled_block_types": ["function_bodies"]]),
+            Example("""
+            func f() {}
+            """, configuration: ["disabled_block_types": ["function_bodies"]]),
+            Example("""
+            deinit {}
+            """, configuration: ["disabled_block_types": ["initializer_bodies"]]),
+            Example("""
+            init() {}
```

Comment:
All examples should be valid Swift. `init`, `deinit` and `defer` need some declaration around them.